src/main/java/frc/robot/RobotContainer.java

```diff
@@ -193,17 +195,19 @@ public void putShuffleboardCommands() {
 
     private void configureBindings() {
         drivetrain.setDefaultCommand(new Drive(drivetrain, joyleft::getY, joyleft::getX, joyright::getX));
-        Command alignSpeaker = CommandUtil.parallel(
+        Command alignSpeaker = CommandUtil.sequence(
                 "Shoot&AlignSpeaker",
                 new SetAlignable(drivetrain, Alignable.SPEAKER),
-                new AdvancedSpinningShot(shooter, () -> drivetrain.getPose(), () -> drivetrain.getAlignable())
+                new ParallelCommandGroup(
+                        new AdvancedSpinningShot(shooter, () -> drivetrain.getPose(), () -> drivetrain.getAlignable()),
+                        new RampFollow(ramp,() -> drivetrain.getAlignable(), () -> drivetrain.getPose())
+                )
         );
         joyLeftButton1.onTrue(alignSpeaker).onFalse(CommandUtil.logged(new SetAlignable(drivetrain, null)));
```

Comment:
Should we stop the shooter and reset the ramp on false?

Reply:
Shooter does stop (command returns false). I was thinking about resetting the ramp but could not decide if it was a good idea. Do you have any reasons to why we would reset it?

Reply:
I don't see what would stop the shooter. Please inform?
Yes, probably good to reset the ramp - one less thing to do before next intake